src/iotjs_binding.c

```diff
@@ -105,16 +105,9 @@ iotjs_jval_t iotjs_jval_create_function(JHandlerType handler) {
 }
 
 
-iotjs_jval_t iotjs_jval_create_error(const char* msg) {
-  return iotjs_jval_create_error_type(JERRY_ERROR_COMMON, msg);
-}
-
-
-iotjs_jval_t iotjs_jval_create_error_type(jerry_error_t type, const char* msg) {
-  iotjs_jval_t jval;
-
-  const jerry_char_t* jmsg = (const jerry_char_t*)(msg);
-  jval = jerry_create_error(type, jmsg);
+iotjs_jval_t iotjs_jval_create_error_without_error_flag(const char* msg) {
```

Comment:
@LaszloLango Do we need to rename `iotjs_jval_create_error` to `iotjs_jval_create_error_without_error_flag`? We always create error that has cleared error flag. We may introduce `iotjs_jval_create_error_without_error_flag` later when we need `iotjs_jval_create_error_with_error_flag`.

Reply:
IMHO using errors without error flags in the whole code base is a bad practice. My motivation was to clarify this part. Do you think we should keep to current logic and never use errors with error flags?

Reply:
I think there is no change in logic for creating error. Previously it created the error without error flag, currently it is same. The only change is name (`iotjs_jval_create_error` to `iotjs_jval_create_error_without_error_flag`). Do you prefer `iotjs_jval_create_error_without_error_flag` since it is more descriptive name for clarification?

Reply:
I think Laszlo prefers to use the error flag logic built-into the JerryScript. The motivation of having a separate throws flag was supporting multiple engines. However now the flag is obsolete, and we can use the error management system of JerryScript.

Reply:
@zherczeg @LaszloLango Still, I don't understand. Another question raises from @zherczeg's comment. `iotjs_jval_create_error_without_error_flag` will create error object using `jerry_create_error`, then decrease reference count by `jerry_value_clear_error_flag`. Is it correct?

Reply:
@glistening there are three ways to get error values.
i) It comes from javascript (`jerry_run` return with an error value)
ii) Thrown by `jerry_create_error` when `JS_CREATE_ERROR` is used
iii) Not thrown, but only created by `iotjs_jval_create_error[_without_error_flag]`.

The third case is only needed (if i see this correctly), when module callback must be called with an error to handle.